### DISCUSSION TITLE
Enlarge image on click

### DISCUSSION THREAD

**Discussion by microfuge** [galaxyproject/training-material#4895] [Ideas]
Hi,

This is only in case you find it useful and just a suggestion.

Right now clicking on an image in the tutorial, the image opens as a separate link. It would be good if the image is enlarged on-click instead.  This will not cause break in the flow while following the tutorial. 
The exact behavior would be like this webpage https://maoschanz.github.io/drawing/gallery.html (if you click on any image in the gallery)

Thanks!

**Comment by hexylena** [comment]
Hi @microfuge thanks for the idea! It is something we could consider but it would require accessibility testing. I note that there is no way to activity the enlargement with keyboard, we would need to implement something for that, but it is indeed something done on other sites quite commonly. The current implementation is not great, so, it is a good idea that we look at others going forward.
